sql/core/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -649,6 +649,14 @@ object SQLConf {
       .doubleConf
       .createWithDefault(0.05)
 
+  val SHOW_STATS_IN_EXPLAIN =
```

Comment:
Then, when the stats are not accurate, will it be the cause of an inefficient plan? If so, why not showing them the number?